conf/log4j.properties

```diff
@@ -181,6 +181,7 @@ log4j.appender.FUSE_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%
 log4j.logger.io.grpc.netty.NettyServerHandler=OFF
 
 # Disable noisy INFO logs from ratis
+log4j.logger.org.apache.ratis.grpc.server.GrpcLogAppender=ERROR
```

Comment:
Do we want to also ignore the WARN level logs for GrpcLogAppender?

Reply:
Yes this line tells log4j to only record `ERROR` level logs and ignore `INFO` or `WARN` level logs.

Reply:
Is there any reason for us to disable the WARN level log for `GrpcLogAppender`? Will it filter out some useful WARN level logs?

Reply:
The main purpose is to suppress the excessive messages from `GrpcLogAppender` when a follower is offline:

```
2020-10-14 17:56:30,594 WARN  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-InstallSnapshotResponseHandler: Failed InstallSnapshot: org.apache.ratis.thirdparty.io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2020-10-14 17:56:30,594 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: followerNextIndex = 0 but logStartIndex = 24044765, notify follower to install snapshot-(t:188, i:24044765)
2020-10-14 17:56:30,594 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: send FaultToleranceScaleEmbeddedJournal1-masters-1_19200->FaultToleranceScaleEmbeddedJournal1-masters-4_19200#0-t212,notify:(t:188, i:24044765)
2020-10-14 17:56:30,595 WARN  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-InstallSnapshotResponseHandler: Failed InstallSnapshot: org.apache.ratis.thirdparty.io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2020-10-14 17:56:30,595 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: followerNextIndex = 0 but logStartIndex = 24044765, notify follower to install snapshot-(t:188, i:24044765)
2020-10-14 17:56:30,595 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: send FaultToleranceScaleEmbeddedJournal1-masters-1_19200->FaultToleranceScaleEmbeddedJournal1-masters-4_19200#0-t212,notify:(t:188, i:24044765)
2020-10-14 17:56:30,595 WARN  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-InstallSnapshotResponseHandler: Failed InstallSnapshot: org.apache.ratis.thirdparty.io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2020-10-14 17:56:30,595 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: followerNextIndex = 0 but logStartIndex = 24044765, notify follower to install snapshot-(t:188, i:24044765)
2020-10-14 17:56:30,595 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: send FaultToleranceScaleEmbeddedJournal1-masters-1_19200->FaultToleranceScaleEmbeddedJournal1-masters-4_19200#0-t212,notify:(t:188, i:24044765)
2020-10-14 17:56:30,596 WARN  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-InstallSnapshotResponseHandler: Failed InstallSnapshot: org.apache.ratis.thirdparty.io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2020-10-14 17:56:30,596 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: followerNextIndex = 0 but logStartIndex = 24044765, notify follower to install snapshot-(t:188, i:24044765)
...
```